.github/workflows/phpunit.yaml

```diff
@@ -0,0 +1,9 @@
+name: PHPUnit
+on: push
+jobs:
+    test:
+        runs-on: ubuntu-latest
+        steps:
+            - uses: actions/checkout@v2
+            - uses: php-actions/composer@v1
+            - uses: php-actions/phpunit@v8
```

Comment:
Pipeline ist still failing. Check why. Consider to simply run /vendor/bin/phpunit instead of using some third party action (see channel-core repo)

Reply:
when remove travis.yml consider to replace the badge in README.md with one from github actions